pkg/skaffold/build/local/prune.go

```diff
@@ -115,7 +115,7 @@ func (p *pruner) runPrune(ctx context.Context, ids []string) error {
 
 	beforeDu, err := p.diskUsage(ctx)
 	if err != nil {
-		logrus.Warnf("Failed to get docker usage info: %v", err)
+		logrus.Debugf("Failed to get docker usage info: %v", err)
```

Comment:
Should we continue with the pruning if the usage failed?  In the case in question, the context was cancelled so we should be returning ASAP.

Reply:
The usage may fail, e.g. if another docker request is in progress. It's ok for us, we need the usage data only for logging and just skip the log message if usage request failed. 

Good point about the early return. Fixed.